src/main/java/org/minions/devfund/katerinaanzoleaga/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,280 @@
+package org.minions.devfund.katerinaanzoleaga.moviedatabase;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * MovieDatabase class.
+ */
+public class MovieDatabase {
+    ArrayList<Movie> movieList;
+    ArrayList<Actor> actorList;
+
+    /**
+     * Constructor starts empty Movies and Actors' list.
+     */
+    MovieDatabase() {
+        this.movieList = new ArrayList<Movie>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)